src/api/auth/BaseAuth.ts

```diff
@@ -31,10 +31,10 @@ export abstract class BaseAuth implements IAuthAdapter {
   ): Promise<IAuthToken> {
     /* wait for tokens */
     return tokenPair.then((tokens: IAuthToken) => requestAdapter.execute(
-      this.buildLoginUrl(projectID), {
+      this.buildLoginUrl(projectID) + "/refresh", {
```

Comment:
If we're using interpolation of strings to create the URL in other methods it's better to keep consistency also here.